src/components/Company.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export class Company extends React.Component {
+    render () {
+        const { company = {} } = this.props;
+
+        return (
```

Comment:
this component don't use state so we can rewrite it in functional manner, because using functions is preferable